tendrl/commons/flows/import_cluster/gluster_help.py

```diff
@@ -90,6 +90,8 @@ def import_gluster(parameters):
     ]['tags']['tendrl-gluster-integration']
     config_data = {"etcd_port": int(NS.config.data['etcd_port']),
                    "etcd_connection": str(NS.config.data['etcd_connection']),
+                   "etcd_username": "root",
```

Comment:
Why are you hardcoding root username when we have a config called etcd_username ?

Reply:
Done. Will take the values from node-agent's conf and set here.